buildenv/jenkins/JenkinsfileBase

```diff
@@ -249,6 +249,14 @@ def setup() {
 				env.STF_BRANCH = params.STF_BRANCH
 			}
 
+			if (params.BUMBLEBENCH_REPO) {
+				env.BUMBLEBENCH_REPO = params.BUMBLEBENCH_REPO
+			}
+
+			if (params.BUMBLEBENCH_BRANCH) {
+				env.BUMBLEBENCH_BRANCH = params.BUMBLEBENCH_BRANCH
+			}
+			
```

Comment:
Prefer not to add another parameter to Jenkinsfilebase, as it sets precedence to add many more for each of the perf and external subdirectories.  

The use case for requiring to point to a specific bumblebench branch is to test new microbenches or changes to existing ones.  This can be accomplished by using ADOPTOPENJDK_REPO/ADOPTOPENJDK_BRANCH and changing lines 34 and 37 of perf/bumbleBench/build.xml to point to the dev repo/branch.